grblas/base.py

```diff
@@ -49,20 +49,16 @@ def update(self, delayed):
 
 class GbContainer:
     # Flag for operations which depend on scalar vs vector/matrix
-    is_scalar = False
+    _is_scalar = False
 
     def __init__(self, gb_obj, dtype):
         if not isinstance(gb_obj, ffi.CData):
             raise TypeError('Object passed to __init__ must be CData type')
         if not isinstance(dtype, dtypes.DataType):
             dtype = dtypes.lookup(dtype)
-
         self.gb_obj = gb_obj
         self.dtype = dtype
 
-    def __invert__(self):
-        return ComplementedMask(self)
```

Comment:
Removing this means a user cannot `(~M).V`. Probably not worth allowing.
If we are removing this, we should remove the ComplementedMask class.

Should we give a useful error message if someone tries to `mask=~M`? A message that masks need to indicate either `.V` or `.S`?

Reply:
Good call on `ComplementedMask` (a coverage report would have hinted at this).  A useful error message here may not hurt.